qiskit_experiments/library/tomography/tomography_experiment.py

```diff
@@ -139,6 +146,15 @@ def __init__(
             target_circuit.append(circuit, range(num_qubits))
         self._circuit = target_circuit
 
+        self._cond_clbits = None
+        if conditional_circuit_clbits is True:
+            conditional_circuit_clbits = self._circuit.clbits
+        if conditional_circuit_clbits:
```

Comment:
This case handles sequences of integers or `Clbit` objects. I would say the main reason to allow for cblits would be if you had some more complicated circuit (maybe something like teleportation/quantum error detection/post-selection etc) with multiple clbits/registers and it may be more error-prone to assume the index position of the bit in the circuit (These experiments do accept circuits with multiple registers)

Reply:
That makes sense